runner/print_version.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
FYI: This script is copied from go-livepeer, so we'll use exactly the same sem versioning as go-livepeer

Reply:
Thanks for letting me know looks good. 👍🏻